cvat/apps/engine/serializers.py

```diff
@@ -79,6 +79,7 @@ class LabelSerializer(serializers.ModelSerializer):
     class Meta:
         model = models.Label
         fields = ('id', 'name', 'color', 'attributes', 'deleted')
+        ordering = ['-id']
```

Comment:
Could you please add a link on documentation about the field? I can see such parameter for Django Models: https://docs.djangoproject.com/en/3.2/ref/models/options/#ordering

Reply:
Or may be on the DRF source code. Because in IAM PR I deleted all such fields.

Reply:
Could not find any reference. So, moved to custom prefetching of related fields.